### ISSUE TITLE
strange path value

### ISSUE THREAD

**Issue by gabyrusu** [bugventure/jsen#72]
Hi, guys! I have the schema below:

```
{
    "$schema": "http://json-schema.org/draft-04/schema#",
    "title": "User",
    "description": "User validation schema",
    "type": "object",
    "messages": {
        "type": "Invalid data type where an object is expected"
    },
    "properties": {
        "first_name": {
            "description": "User first name",
            "type": "string",
            "minLength": 2,
            "maxLength": 50,
            "invalidMessage": "The first name is not valid",
            "requiredMessage": "The first name is required"
        },
        "last_name": {
            "description": "User last name",
            "type": "string",
            "minLength": 2,
            "maxLength": 50,
            "invalidMessage": "The last name is not valid",
            "requiredMessage": "The last name is required"
        },
        "email": {
            "description": "User email address",
            "type": "string",
            "format": "email",
            "invalidMessage": "The email is not valid",
            "requiredMessage": "The email is required"
        }
    },
    "required": [
        "first_name",
        "last_name",
        "email"
    ]
}
```

When trying to validate this object: {first_name: "A", last_name: "B", email: "cd"}
The validation errors are:

```
[
    {
        "keyword": "minLength",
        "message": "The first name is not valid",
        "path": ""first_name"]" #incorrect
    },
    {
        "keyword": "minLength",
        "message": "The last name is not valid",
        "path": ""last_name"]" #incorrect
    },
    {
        "keyword": "format",
        "message": "The email is not valid",
        "path": "email" #correct
    }
]
```

I'm not sure why the path value is ""first_name"]" instead of "first_name".
What am I doing wrong?

PS: I'm using jsen v0.6.3.

**Comment by gabyrusu** [comment]
So it seams like the property names must be camel-cased.

**Comment by gabyrusu** [comment]
I see that this behavior appears only if the property is invalid. on required the path is "first_name" as it should so I believe this could be a bug. Can you check this through?

Thank you!

**Comment by bugventure** [comment]
Thanks for reporting this, @gabyrusu. Will be fixed in v0.6.4
